cypress/integration/sitewide-banner.js

```diff
@@ -99,8 +129,14 @@ describe('Site Wide Banner', () => {
     const user = userFactory();
 
     // Log in & visit the campaign pitch page:
-    cy.authVisitCampaignWithoutSignup(user, exampleCampaign);
 
+    cy.mockGraphqlOp('VoterRegSiteWideBannerQuery', {
```

Comment:
Ahh I think this test is failing because we need to update our mock query per the rename we did
```suggestion
    cy.mockGraphqlOp('VoterRegSitewideBannerQuery', {
```

Same with the ones in the earlier tests!